docs/content/drive.md

```diff
@@ -202,7 +202,9 @@ actively logged-in users, for example build machines.
 To use a Service Account instead of OAuth2 token flow, enter the path
 to your Service Account credentials at the `service_account_file`
 prompt during `rclone config` and rclone won't use the browser based
-authentication flow.
+authentication flow. If you'd rather stuff the contents of the file
```

Comment:
That should probably say "If you'd rather stuff the contents of the credentials file into the rclone config file you can put it under the `service_account_credentials` key, or set the equivalent environment variable."